packages/slo-stat-bot/test/fixtures/config/valid-config.yml

```diff
@@ -0,0 +1 @@
+randomBoolean: true
```

Comment:
You can delete this file, it's just part of the generation of a probot in case you want to use a config file.